### PR TITLE
BGDIINF_SB-2282: Removed kml service and gstyles services

### PR DESCRIPTION
**Pull request by ltshb**

### CODE REVIEW

File: chsdi/__init__.py [hunk 1]
```diff
@@ -103,40 +103,10 @@ def datetime_adapter(obj, request):
...
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-    config.add_route('glstyles', '/gl-styles/{id}', request_method=('GET', 'POST', 'DELETE'))
-    add_cors_route(config,
-                   '/gl-styles/{id}',
-                   'glstyles',
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-
-    config.add_route('adminkml', '/admin/kml')
```
Comment (procrastinatio) [inline chsdi/__init__.py:139]: Do we also remove the admin interface to KML?

**Review by procrastinatio — APPROVED** [review]
So we also silently remove the admin KML interface ? Otherwise OK

File: chsdi/__init__.py (hunk 1 above)
Comment (boecklic) [reply chsdi/__init__.py:139]: Yup, we can remove that

**Review by boecklic — APPROVED** [review]

File: apache/wsgi-py3.conf.in [hunk 2]
```diff
@@ -158,10 +158,11 @@ Header always set Access-Control-Allow-Headers "x-requested-with, Content-Type,
 # kml file downloads
 AliasMatch ^${APACHE_ENTRY_PATH}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${KML_TEMP_DIR}/$1
 
-# KML admin
+# KML admin TODO KML
 RewriteCond %{HTTP:X-Forwarded-Proto} !https
 RewriteRule ^${APACHE_ENTRY_PATH}/admin         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
```
Comment (ltshb) [inline apache/wsgi-py3.conf.in:163]: @procrastinatio @boecklic can we remove those lines  ? Are they only for kml or other purpose ?

File: apache/wsgi-py3.conf.in [hunk 3]
```diff
@@ -158,10 +158,11 @@ Header always set Access-Control-Allow-Headers "x-requested-with, Content-Type,
...
 
+# TODO KML
 <LocationMatch ^${APACHE_ENTRY_PATH}/(admin/kml|static/info.json)>
     AuthType Basic
     AuthName "Your IWI credential"
```
Comment (ltshb) [inline apache/wsgi-py3.conf.in:168]: @procrastinatio @boecklic same here the LocationMatch can be removed ? What is the `static/info.json` ? I guess this needs to be kept ?

File: apache/wsgi-py3.conf.in (hunk 2 above)
Comment (procrastinatio) [reply apache/wsgi-py3.conf.in:163]: Yes, the `admin`path may be removed

File: apache/wsgi-py3.conf.in (hunk 3 above)
Comment (procrastinatio) [reply apache/wsgi-py3.conf.in]: `info.json` is a file with version info (build version, date, etc.)

**Comment by ltshb** [comment]
@boecklic @procrastinatio I updated the readme as well as the Apache Configuration could you check it again before I merge (see last 2 commits) Thanks.

File: apache/wsgi-py3.conf.in (hunk 3 above)
Comment (procrastinatio) [reply apache/wsgi-py3.conf.in]: If you remove this line, remove everything concerning `info.json`

**Review by procrastinatio — APPROVED** [review]
Looks OK (in theory)

**Comment by ltshb** [comment]
I'll merge hopefully not breaking anything with the apache conf...

**Pull request merged**